screenpy/pacing.py

```diff
@@ -31,7 +31,7 @@ def function_should_log_none(func: Function) -> bool:
     return False
 
 
-def act(title: str, gravitas: Optional[str] = None) -> Callable[[Function], Function]:
+def act(title: str, gravitas: str | None = None) -> Callable[[Function], Function]:
```

Comment:
It might be a good idea, what do you think about calling it something thematic?

```python
Narrative = Callable[[Function], Function]  # ?
Tale = Callable[[Function], Function]  # ?
Yarn = Callable[[Function], Function]  # ?
...
```